.circleci/config.yml

```diff
@@ -133,6 +148,57 @@ jobs:
       - run:
           name: Run pylint
           command: ./tools/pylint.sh
+  setup-test-reporter:
+    docker:
+      - image: cimg/base:stable
+    resource_class: small
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Install CodeClimate Test Reporter
+          command: |
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+            chmod +x ./cc-test-reporter
+      - run:
+          name: Notify CodeClimate of a pending report
+          command: ./cc-test-reporter before-build
+      - persist_to_workspace:
+          root: .
+          paths:
+            - cc-test-reporter
+  test:
+    docker:
+      - image: cimg/python:3.9.16
+      - image: cimg/postgres:14.1
+        environment:
+          POSTGRES_USER: integreat
+          POSTGRES_DB: integreat-compass
+          POSTGRES_PASSWORD: password
+    resource_class: small
+    parallelism: 2
+    steps:
+      - run:
+        name: Enable virtual environment
+        command: echo "source .venv/bin/activate" >> $BASH_ENV
```

Comment:
```suggestion
          name: Enable virtual environment
          command: echo "source .venv/bin/activate" >> $BASH_ENV
```
The indentation here is incorrect. The CircleCI workflow is not run:
https://app.circleci.com/pipelines/github/digitalfabrik/integreat-compass?branch=setup-ci-testing

![Screenshot 2023-06-30 at 14-23-14 Pipelines - digitalfabrik_integreat-compass](https://github.com/digitalfabrik/integreat-compass/assets/16021269/dab84461-caa8-400b-9939-a23e79dd83c4)